Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
Read 2948 events.
Traceback (most recent call last):
  File "./sts/happensbefore/hb_graph.py", line 1533, in <module>
    main.run()
  File "./sts/happensbefore/hb_graph.py", line 1296, in run
    self.graph.load_trace(self.filename)
  File "./sts/happensbefore/hb_graph.py", line 544, in load_trace
    self.add_event(event)
  File "./sts/happensbefore/hb_graph.py", line 531, in add_event
    handlers.get(event.type, _handle_default)(event)
  File "./sts/happensbefore/hb_graph.py", line 502, in _handle_HbMessageHandle
    self._update_shadow_tables(event)
  File "./sts/happensbefore/hb_graph.py", line 442, in _update_shadow_tables
    self.shadow_tables[event.dpid].apply_event(event)
  File "/home/ubuntu/www/public_html/jenkins/latest_sts_build/sts/happensbefore/hb_shadow_table.py", line 144, in apply_event
    assert compare_flow_table(self.table, op.flow_table)
AssertionError
